README.md

```diff
@@ -8,16 +8,20 @@ Mobility aggregation service based on the [General Bikeshare Feed Specification
 
         lamassu:
             providers:
-              
-                systemName: My System
+              - systemId: mysystem
                 operatorId: MFM:Operator:myoperator
                 operatorName: My operator
                 codespace: MFM
                 url: https://myfavoritegbfsfeeed.com/gbfs.json
-                language: en
-
+                language: en             
+                
 This will use the GBFS auto-discovery at `url` and poll `en` language feeds.
 
+NOTE: codespace is relevant when aggregating entities across systems, to avoid conflicts between IDs. 
+Every ID (except systemId, which (by GBFS convention) should already be globally unique) is prefixed by this codespace.
+Every feed should be assigned its own unique codespace, even when published by the same operator.
```

Comment:
Actually, at Entur we assign a codespace per data provider (i.e. operator) and we assume that operators are able to maintain data separation between their own feeds. Our concern is data separation across data providers.

Reply:
Adapted notes accordingly.